### DISCUSSION TITLE
MRF V2.0.0 Plan name

### DISCUSSION THREAD

**Discussion by asvbarnes** [CMSgov/hospital-price-transparency#77] [Q&A] [answered]
If a payer contract covers all plans and does not specify a specific plan name under the one contract that contains one fee schedule and/or rate sheet, Is it acceptable to encode "All Plans" as the Payer_plan name? For example, the field may look like this: standard_charge|ABC_Health_Insurance|All_Commercial_Plans|methodology.

**Comment by RidenourK** [comment, marked as answer]
@asvbarnes Thank you for your question. 

Yes. As indicated in the CY 2024 OPPS/ASC Final Rule, in instances where the hospital, within the contract with the third-party payer, has negotiated the same payer-specific charges for a category of plans, the hospital may indicate the category of plans rather than the specific name(s) of the plans. Please see the discussion beginning at [88 FR 82093](https://www.federalregister.gov/d/2023-24293/p-4944).
